scripts/package.ts

```diff
@@ -8,7 +8,7 @@ export function build(
   prettierVersion
 ) {
   try {
-    execSync('npx nx run-many --target=build --all --parallel', {
```

Comment:
Why is this not parallel anymore? Is it because CI doesn't run well with parallel?

Reply:
Parallel builds fail randomly on Windows 🤷 We could use the `--parallel` in case of Linux, but since there is no big time difference, I preferred to keep it simple.